special-interest-groups/sig-planner/roles-and-organization-management.md

```diff
@@ -0,0 +1,64 @@
+# Planner SIG Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the Planner SIG is to have
+the **Active Contributor** community role in the planner/statistics package.
+
+### Promotion
+
+#### From `Contributor` to `Active Contributor`
+
+Requirements:
+
+* Have at least 3 PRs merged in the planner/statistics package within one year.
+* Nominated by at least 2 Planner SIG Reviewers, Committer or Maintainer.
+
+#### From `Active Contributor` to `Reviewer`
+
+Requirements:
+
+* Have at least 10 PRs merged in the planner/statistics package within one year.
+* Nominated by at least 2 Committers or 2 Maintainers.
+
+#### From `Reviewer` to `Committer`
+
+Requirements:
+
+* Have at least 30 PRs merged in the planner/statistics package within one year.
+* Have reviewed at least 15 PRs in the planner/statistics package within one year.
+* Nominated by at least 2 Committers or Maintainers.
```

Comment:
The promotion for reviewer to committer should not conflict with the [TiKV community membership](https://github.com/tikv/community/blob/master/community-membership.md), committer must have approval by 2 maintainers or above

Reply:
It's copied from sig-expr. So i think @qw4990 should fix that one too?